manager-dashboard/.env.example

```diff
@@ -0,0 +1,10 @@
+# REACT_APP_SENTRY_DSN=
+# REACT_APP_ENVIRONMENT=
+
+REACT_APP_FIREBASE_API_KEY=
+REACT_APP_FIREBASE_AUTH_DOMAIN=
+REACT_APP_FIREBASE_DATABASE_URL=
+REACT_APP_FIREBASE_PROJECT_ID=
+REACT_APP_FIREBASE_STORAGE_BUCKET=
+REACT_APP_FIREBASE_MESSAGING_SENDER_ID=
+REACT_APP_FIREBASE_APP_ID=
```

Comment:
here you define another .env file. Would it make sense to use the existing one or was there any particular reason to have it like this?

Reply:
These are for specific components. Maybe we can discuss further on this.